frontend/src/app/hierarchy-viewer/hierarchy-viewer.component.html

```diff
@@ -1,23 +1,32 @@
 <div class="header-container">
     <mat-form-field class="select" appearance="outline" *ngIf="hierarchies$ | async as hierarchies">
         <mat-select [value]="selectedHierarchy?.id ?? ' '" (selectionChange)="onSelect($event)">
-            <mat-option *ngFor="let hierarchy of hierarchies" [value]="hierarchy.id">{{hierarchy.name}}</mat-option>
+            <mat-option *ngFor="let hierarchy of hierarchies" [value]="hierarchy.id">{{ hierarchy.name }}</mat-option>
         </mat-select>
     </mat-form-field>
-    <button mat-raised-button color="accent" class="header-button">
+    <button mat-raised-button color="accent" (click)="createHierarchyDialog()" class="header-button">
         <mat-icon>add</mat-icon>
         Create Hierarchy
     </button>
-    <button mat-raised-button color="warn" class="header-button">
-        <mat-icon>delete</mat-icon>
-        Delete Hierarchy
+
+    <ng-container *ngIf="selectedHierarchy">
+        <button mat-raised-button color="warn" class="header-button" (click)="deleteHierarchyDialog()">
+            <mat-icon>delete</mat-icon>
+            Delete Hierarchy
+        </button>
+    </ng-container>
+
+    <button mat-raised-button color="accent" class="header-button" (click)="importExportHierarchyDialog()">
+        <mat-icon>import_export</mat-icon>
+        Import/Export Hierarchy
     </button>
 
     <!-- This is a short term hack to save RRRHierarchy into the db -->
-    <button mat-raised-button color="accent" class="header-button" (click)="createHierarchy()">
```

Comment:
You can delete this button and the importRRRHierarchy() function. We can just use your import tool from now on to import the RRRHierarchyPost.json from src/assests/staticFiles.